diff --git a/lib/compress/clevels.h b/lib/compress/clevels.h
index c18da465..04f918d5 100644
--- a/lib/compress/clevels.h
+++ b/lib/compress/clevels.h
@@ -1,15 +1,5 @@
-/*
- * Copyright (c) Meta Platforms, Inc. and affiliates.
- * All rights reserved.
- *
- * This source code is licensed under both the BSD-style license (found in the
- * LICENSE file in the root directory of this source tree) and the GPLv2 (found
- * in the COPYING file in the root directory of this source tree).
- * You may select, at your option, one of the above-listed licenses.
- */
-
 #ifndef ZSTD_CLEVELS_H
-#define ZSTD_CLEVELS_H
+#define ZSTD_CLEVELS_H  /* Block default definitions */
 
 #define ZSTD_STATIC_LINKING_ONLY  /* ZSTD_compressionParameters  */
 #include "../zstd.h"
@@ -25,29 +15,29 @@ __attribute__((__unused__))
 static const ZSTD_compressionParameters ZSTD_defaultCParameters[4][ZSTD_MAX_CLEVEL+1] = {
 {   /* "default" - for any srcSize > 256 KB */
     /* W,  C,  H,  S,  L, TL, strat */
-    { 19, 12, 13,  1,  6,  1, ZSTD_fast    },  /* base for negative levels */
-    { 19, 13, 14,  1,  7,  0, ZSTD_fast    },  /* level  1 */
-    { 20, 15, 16,  1,  6,  0, ZSTD_fast    },  /* level  2 */
-    { 21, 16, 17,  1,  5,  0, ZSTD_dfast   },  /* level  3 */
-    { 21, 18, 18,  1,  5,  0, ZSTD_dfast   },  /* level  4 */
-    { 21, 18, 19,  3,  5,  2, ZSTD_greedy  },  /* level  5 */
-    { 21, 18, 19,  3,  5,  4, ZSTD_lazy    },  /* level  6 */
-    { 21, 19, 20,  4,  5,  8, ZSTD_lazy    },  /* level  7 */
-    { 21, 19, 20,  4,  5, 16, ZSTD_lazy2   },  /* level  8 */
-    { 22, 20, 21,  4,  5, 16, ZSTD_lazy2   },  /* level  9 */
-    { 22, 21, 22,  5,  5, 16, ZSTD_lazy2   },  /* level 10 */
-    { 22, 21, 22,  6,  5, 16, ZSTD_lazy2   },  /* level 11 */
-    { 22, 22, 23,  6,  5, 32, ZSTD_lazy2   },  /* level 12 */
-    { 22, 22, 22,  4,  5, 32, ZSTD_btlazy2 },  /* level 13 */
-    { 22, 22, 23,  5,  5, 32, ZSTD_btlazy2 },  /* level 14 */
-    { 22, 23, 23,  6,  5, 32, ZSTD_btlazy2 },  /* level 15 */
-    { 22, 22, 22,  5,  5, 48, ZSTD_btopt   },  /* level 16 */
-    { 23, 23, 22,  5,  4, 64, ZSTD_btopt   },  /* level 17 */
-    { 23, 23, 22,  6,  3, 64, ZSTD_btultra },  /* level 18 */
-    { 23, 24, 22,  7,  3,256, ZSTD_btultra2},  /* level 19 */
-    { 25, 25, 23,  7,  3,256, ZSTD_btultra2},  /* level 20 */
-    { 26, 26, 24,  7,  3,512, ZSTD_btultra2},  /* level 21 */
-    { 27, 27, 25,  9,  3,999, ZSTD_btultra2},  /* level 22 */
+    { 18/*19*/, 12, 13,  1,  6,  1, ZSTD_fast    },  /* base for negative levels */
+    { 18/*19*/, 13, 14,  1,  7,  0, ZSTD_fast    },  /* level  1 (ATG modified, 256K window)*/
+    { 18/*20*/, 15, 16,  1,  6,  0, ZSTD_fast    },  /* level  2 (ATG modified, 256K window)*/
+    { 18/*21*/, 16, 17,  1,  5,  0, ZSTD_dfast   },  /* level  3 (ATG modified, 256K window)*/
+    { 18/*21*/, 18, 18,  1,  5,  0, ZSTD_dfast   },  /* level  4 (ATG modified, 256K window)*/
+    { 18/*21*/, 18, 19,  3,  5,  2, ZSTD_greedy  },  /* level  5 (ATG modified, 256K window)*/
+    { 18/*21*/, 18, 19,  3,  5,  4, ZSTD_lazy    },  /* level  6 (ATG modified, 256K window)*/
+    { 18/*21*/, 19, 20,  4,  5,  8, ZSTD_lazy    },  /* level  7 (ATG modified, 256K window)*/
+    { 18/*21*/, 19, 20,  4,  5, 16, ZSTD_lazy2   },  /* level  8 (ATG modified, 256K window)*/
+    { 18/*22*/, 20, 21,  4,  5, 16, ZSTD_lazy2   },  /* level  9 (ATG modified, 256K window)*/
+    { 18/*22*/, 21, 22,  5,  5, 16, ZSTD_lazy2   },  /* level 10 (ATG modified, 256K window)*/
+    { 18/*22*/, 21, 22,  6,  5, 16, ZSTD_lazy2   },  /* level 11 (ATG modified, 256K window)*/
+    { 18/*22*/, 22, 23,  6,  5, 32, ZSTD_lazy2   },  /* level 12 (ATG modified, 256K window)*/
+    { 18/*22*/, 22, 22,  4,  5, 32, ZSTD_btlazy2 },  /* level 13 (ATG modified, 256K window)*/
+    { 18/*22*/, 22, 23,  5,  5, 32, ZSTD_btlazy2 },  /* level 14 (ATG modified, 256K window)*/
+    { 18/*22*/, 23, 23,  6,  5, 32, ZSTD_btlazy2 },  /* level 15 (ATG modified, 256K window)*/
+    { 18/*22*/, 22, 22,  5,  5, 48, ZSTD_btopt   },  /* level 16 (ATG modified, 256K window)*/
+    { 18/*23*/, 23, 22,  5,  4, 64, ZSTD_btopt   },  /* level 17 (ATG modified, 256K window)*/
+    { 18/*23*/, 23, 22,  6,  3, 64, ZSTD_btultra },  /* level 18 (ATG modified, 256K window)*/
+    { 18/*23*/, 24, 22,  7,  3,256, ZSTD_btultra2},  /* level 19 (ATG modified, 256K window)*/
+    { 18/*25*/, 25, 23,  7,  3,256, ZSTD_btultra2},  /* level 20 (ATG modified, 256K window)*/
+    { 18/*26*/, 26, 24,  7,  3,512, ZSTD_btultra2},  /* level 21 (ATG modified, 256K window)*/
+    { 18/*27*/, 27, 25,  9,  3,999, ZSTD_btultra2},  /* level 22 (ATG modified, 256K window)*/
 },
 {   /* for srcSize <= 256 KB */
     /* W,  C,  H,  S,  L,  T, strat */
